src/Interface/Modules/Render/ViewScene.h

```diff
@@ -321,8 +323,18 @@ namespace SCIRun {
       bool                                  invertZoom_                   {};
       bool                                  shiftdown_                    {false};
       bool                                  mouseButtonPressed_           {false};
+      bool                                  frameIsFinished_              {false};
       Graphics::Datatypes::WidgetHandle     selectedWidget_;
+      Graphics::Datatypes::WidgetHandle     previousSelectedWidget_;
+      glm::mat4                             previousCameraTransform_      {0.0};
+      std::chrono::_V2::system_clock::time_point timeWidgetColorRestored_ {};
```

Comment:
These variables definitely should not be in the main header, but in an impl class in the cc file only. They are probably what break the mac and windows builds. Please fix

Reply:
I made a new class in ViewScene for checking previous widget selection information.